doc/cprover-manual/cbmc-unwinding.md

```diff
@@ -161,7 +161,9 @@ to the later assertion, use the option
 This option will allow paths that execute loops only partially, enabling
 a counterexample for the assertion above even for small unwinding
 bounds. The disadvantage of using this option is that the resulting path
-may be spurious, that is, it may not exist in the original program.
+may be spurious, that is, it may not exist in the original program. If
+`--unwinding-assertions` is also used and all unwinding assertions pass,
```

Comment:
> If `--unwinding-assertions` is also used [together with --partial-loops] and all unwinding assertions pass

I can't quite see the example where this would happen.  Does the following statment hold? _Whenever a failing assertion becomes reachable due to the use of --partial-loops, there must be a failing unwinding assertion._ ❓

Reply:
Fair point. I have now reworded the text as follows:
```
If `--unwinding-assertions` is also used, and the particular counterexample
trace does not include a report of a violated unwinding assertion, then that
counterexample is not impacted by insufficient loop unwinding.
```